src/pages/attend-events.js

```diff
@@ -0,0 +1,70 @@
+import cx from 'classnames';
+import SEO from '../components/Seo';
+import { Button, Video } from '@newrelic/gatsby-theme-newrelic';
+import PageLayout from '../components/PageLayout';
+import ExternalLink from '../components/ExternalLink';
+import FeatherIcon from '../components/FeatherIcon';
+import nerdDays from '../images/nerd-days/nerd-days.png';
+import styles from './nerd-days.module.scss'
+
+const EventLandingPage = () => {
+  return (
+    <>
+      <SEO />
+      <PageLayout type={PageLayout.TYPE.SINGLE_COLUMN}>
+        <PageLayout.Header title="Developer Events" />
+        <PageLayout.Content>
+          <section className={cx(styles.section, styles.twoColumn)}>
+            <div>
+              <h3>
+                Developer Events <br />
+              </h3>
+              <p>
+                There are a lot of developer events out there, so we've curated a list of events we're hosting or sponsoring that are dedicated to developers, engineers, problem solvers, and builders like you.<br><br>
```

Comment:
```suggestion
                There are a lot of developer events out there, so we've curated a list of events we're hosting or sponsoring that are dedicated to developers, engineers, problem solvers, and builders like you.<br /><br />
```

Found some more br tags that needed to be self-closing.